std/digest/crc.d

```diff
@@ -155,15 +155,17 @@ struct CRC32
         /**
          * Use this to feed the digest with data.
          * Also implements the $(REF isOutputRange, std,range,primitives)
-         * interface for $(D ubyte) and $(D const(ubyte)[]).
+         * interface for single elements or array of const elements of type
+         * `ubyte`, `byte` and `char`.
          */
-        void put(scope const(ubyte)[] data...) @trusted pure nothrow @nogc
+        void put(T)(scope const(T)[] data...) @trusted @nogc
```

Comment:
Templates should almost never be tagged with `@trusted`, because they get their attributes inferred. What should happen is that a unittest that passes safe args to it should be tagged `@safe` which will then check that the template itself is safe.